evals/cybench/task.py

```diff
@@ -0,0 +1,42 @@
+from collections.abc import Callable
+from pathlib import Path
+from textwrap import dedent
+
+from dataset import create_dataset
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.scorer import includes
+from inspect_ai.solver import Plan, basic_agent, plan, system_message
+from inspect_ai.tool import bash, python
+
+
+@task
+def ctf(
+    challenges_dir: Path | list[Path] | None = None,
+    filters: Callable[[Sample], bool] | list[Callable[[Sample], bool]] | None = None,
+    plan: Plan | None = None,
+    max_attempts: int = 3,
+    max_messages: int = 15,
+) -> Task:
+    return Task(
+        dataset=create_dataset(challenges_dir=challenges_dir, filters=filters),
+        plan=plan or default_plan(max_attempts=max_attempts),
+        scorer=includes(),
+        max_messages=max_messages,
+    )
+
+
+@plan
```

Comment:
New school is `@solver` and `default_agent()` for this (the task should also take a `solver` argument rather than a plan)